sentences/cs/cover_HassCloseCover.yaml

```diff
@@ -1,13 +1,15 @@
-language: "cs"
+language: cs
```

Comment:
this is being changed from cs to "cs" and back already multiple times, is there some final guideline which fotmat we should keep?